var footlbotest_leaveMsgUI_leaveMsgUI002_leaveMsgUI002_20190815_173731038 = {"testcase": "footlbotest.leaveMsgUI.leaveMsgUI002.leaveMsgUI002", "steps": [{"name": "\u6d4b\u8bd5\u7528\u4f8b\u521d\u59cb\u5316\u6b65\u9aa4", "start_time": "2019-08-15 17:37:14", "logs": [{"timestamp": "2019-08-15 17:37:29", "level": 60, "message": "run_test\u6267\u884c\u5931\u8d25", "record": {"traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\liubo\\AppData\\Local\\Programs\\Python\\Python37\\lib\\site-packages\\testbase\\testcase.py\", line 701, in _thread_run\n    getattr(self._testcase, it)()\n  File \"C:\\Users\\liubo\\footlbotestproj\\footlbotest\\leaveMsgUI\\leaveMsgUI002.py\", line 44, in run_test\n    driver.find_element_by_id(\"Name12\").send_keys(\"test\")\n  File \"C:\\Users\\liubo\\AppData\\Local\\Programs\\Python\\Python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 360, in find_element_by_id\n    return self.find_element(by=By.ID, value=id_)\n  File \"C:\\Users\\liubo\\AppData\\Local\\Programs\\Python\\Python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 978, in find_element\n    'value': value})['value']\n  File \"C:\\Users\\liubo\\AppData\\Local\\Programs\\Python\\Python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\liubo\\AppData\\Local\\Programs\\Python\\Python37\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\nselenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: [id=\"Name12\"]\n\n"}, "attachments": {}}, {"timestamp": "2019-08-15 17:37:29", "level": 20, "message": "testOver", "record": {}, "attachments": {}}], "succeed": false, "end_time": ["2019-08-15 17:37:31"]}], "failed_info": "", "succeed": false, "start_time": "2019-08-15 17:37:14", "end_time": "2019-08-15 17:37:31"}